src/App.js

```diff
@@ -3,7 +3,7 @@ import { BrowserRouter as Router, Route } from 'react-router-dom';
 
 import './App.css'
 
-import Dashboard from './components/Dashboard'; 
+import Dashboard from './components/dashboard/Dashboard'; 
```

Comment:
Having this all in its own directory really helps make sense of what's going on (and where)